### PR TITLE
Acceleration component

### PR DESCRIPTION
**Pull request by Schtekt**
Acceleration implemented in Game project.

A test exists in the default scene to show off gravity. This could or maybe even should be removed before merge.

Note that the "player" bounces a bit in the ground when landing. This is because of how the physical updates differ from the rendering updates. The Renderer manages to draw the object before the physical position has been applied to the rendering position.

### CODE REVIEW

File: StortSpelProjekt/Game/src/Components/AccelrationComponent.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+void component::AccelrationComponent::SetAccelrationDirection(DirectX::XMFLOAT3 dir)
+{
+	DirectX::XMStoreFloat3(&m_AccDir, DirectX::XMVector3Normalize(DirectX::XMLoadFloat3(&dir)));
+}
+
+void component::AccelrationComponent::SetAccelrationSpeed(float speed)
+{
+	m_AccSpeed = speed;
+}
```
Comment (jocke1995) [inline StortSpelProjekt/Game/src/Components/AccelrationComponent.cpp]: Misspelling of AccelerationComponent?

File: StortSpelProjekt/Game/src/Components/AccelrationComponent.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#ifndef ACCELRATIONCOMPONENT_H
+#define ACCELRATIONCOMPONENT_H
```
Comment (jocke1995) [inline StortSpelProjekt/Game/src/Components/AccelrationComponent.h]: Spelling is wrong

**Review by jocke1995 — DISMISSED** [review]
Well done! I spotted a misspeling of "Acceleration", Change that, remove your test in main and then merge.

**Review by jocke1995 — APPROVED** [review]
Well done

**Review by jakkeez — APPROVED** [review]

**Pull request merged**